.github/workflows/wheels.yml

```diff
@@ -0,0 +1,76 @@
+# Workflow to build and test wheels
+name: Wheel builder
+
+on:
+  schedule:
+    # Nightly build at 3:42 A.M.
+    - cron: "42 3 */1 * *"
+  push:
+    tags:
+      # Stable release build
+      - "[0-9]+.[0-9]+.[0-9]+"
+  pull_request:
+    branches:
+      - master
+      - "[0-9]+.[0-9]+.X"
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest, macos-latest]
```

Comment:
I _think_ we build the wheels on the oldest ubuntu we support, not the latest, or at least the CI used to be like that. Has that changed? Does it matter?

Reply:
On Linux wheels are built inside the manylinux docker, as far as I understand, the distribution of the host shouldn't matter.

Reply:
IIUC, the distribution of the host is for `cibuildwheel` to know the operating system for which the wheels should be built. Actually, the Linux wheels are built inside a `manylinux` container.

Nevertheless, I have noticed that, by default, the wheels are built for `manylinux2010` systems and `scikit-learn` can be built against `manylinux1`. I will change this.

Reply:
> Nevertheless, I have noticed that, by default, the wheels are built for manylinux2010 systems

I think it's a good thing, but opened https://github.com/MacPython/scikit-learn-wheels/issues/62 just in case/